README.md

```diff
@@ -276,6 +276,7 @@ spec:
 
 | Field | Description |
 |-------|-------------|
-| Status | the binding status of Service Binding Request |
+| BindingStatus | The binding status of Service Binding Request |
+| Secret | The name of the intermediate secret |
```

Comment:
Thanks for this @DhritiShikhar . Could you also include a CR yaml in the readme which has the `status` populated?

Reply:
@sbose78 Yes. I am planning to do that. But I don't want to do it as a part of this PR. I will raise another PR for the changes in README.